engine/class_modules/sc_hunter.cpp

```diff
@@ -1290,6 +1297,8 @@ struct hunter_action_t: public Base
     ab::apply_affecting_aura( p -> tier_set.tww_s1_mm_2pc );
     ab::apply_affecting_aura( p -> tier_set.tww_s1_mm_4pc );
     ab::apply_affecting_aura( p -> tier_set.tww_s1_sv_2pc );
```

Comment:
these calls would set up passive auras to modify base values of spell damage etc, but these will be dynamic effects based on buffs, so there's nothing in the tier set driver spelldata to pick up and apply here
you'll need to add appropriate mods in the multiplier calculations below appropriate to the buff, such as winning streak has a direct damage modifier